server/channels/app/platform/service.go

```diff
@@ -381,6 +381,7 @@ func (ps *PlatformService) ShutdownConfig() error {
 
 func (ps *PlatformService) SetTelemetryId(id string) {
 	ps.telemetryId = id
+	ps.regenerateClientConfig()
```

Comment:
How often will this be called? Is there any potential issue to regenerate the client config that often? (Not sure if this will happen often or not)

Reply:
This theoritically needs to be called only once (`PlatformService.SetTelemetryId`), in any case we need to re-compute client config whenever `telemetryId` gets changed.